src/Shapeshifter.WindowsDesktop/Controls/Window/ViewModels/Interfaces/ISettingsViewModel.cs

```diff
@@ -9,6 +9,8 @@ public interface ISettingsViewModel: INotifyPropertyChanged, ISingleInstance
     {
         bool StartWithWindows { get; set; }
 
+		bool QuietMode { get; set; }

```

Comment:
Very nice, but with boolean expressions consider "UseQuietMode" or "IsQuietModeEnabled" which I believe is a better name for a boolean.

Reply:
Makes sense, done!